src/k8s-engine/lima.ts

```diff
@@ -1416,7 +1416,7 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
         // Allow the log process to exit if we're stopping
         return;
       }
-      if (![K8s.State.STARTING, K8s.State.STARTED].includes(this.state)) {
+      if (![K8s.State.STARTING, K8s.State.STARTED, K8s.State.DISABLED].includes(this.state)) {
```

Comment:
@mook-as can you take a look at this and confirm if this is right behavior for the disabled state?

Reply:
No, if state is `DISABLED`, then we should stop following logs if it was previously started (because there's no more logs to follow).